Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php

```diff
@@ -35,6 +35,9 @@
         'assertNotSame' => 'fixAssertNegative',
     );
 
+    /**
+     * @param array<string, bool> $usingMethods
```

Comment:
we can't use phpdoc of this format because it's not supported by php-cs-fixer, no is it widely accepted as valid syntax